src/services/open_ai/query_openai.py

```diff
@@ -164,12 +167,13 @@ def get_prep_message(ctx : Context, messenger, is_final : bool) -> Dict[str, str
 When providing a final answer, use this exact format, and do not deviate.
 IMPORTANT: ALWAYS wrap your final answer with <yair1xigoresponse> tags, and in JSON format.
 
-Today's date is {current_date}.
+Today's full date time is {current_date}.
 For up-to-date information about people, stocks and world events, ALWAYS use one of the tools available to you and DO NOT provide an answer.
 For fiction requests, use your knowledge and creativity to answer.
 If human request has no context of time, assume he is referring to current time period.
 All tools provided have real-time access to the internet; do not reply that you have no access to the internet, unless you have attempted to invoke the SEARCH tool first. Additionally, do not invoke a tool if the required TOOL_INPUT is unknown, vague, or not provided. Always follow the IMPORTANT note in the tool description.
 If you have missing data and ONLY if you cannot use the tools provided to fetch it, try to estimate; in these cases, let the user know your answer is an estimate.
+You are able to set reminders and when you are asked to do it you will invoke the ALERT tool.
```

Comment:
1. Was this needed for R1X to accept reminders?
2. This needs to clearly state reminder has to be in relative time.

Reply:
Need to check what happens in 3.5.

Put a couple of tests in test/data, then use local-test to check.